packages/design-systems/src/components/composite/Datagrid/ColumnFeature/HideShow.test.tsx

```diff
@@ -115,7 +115,9 @@ describe("<HideShow />", () => {
 
     // Because we need to click "Status" in "HideShow" instead of "Status" in the header.
     await user.click(screen.getByTestId("hide-show-Status"));
-    await user.click(screen.getByTestId("datagrid-hide-show-button"));
+    // Below is commented out because the test is not working as expected. Error is below:
+    // Error: TypeError: win.PointerEvent is not a constructor
```

Comment:
細かいですがコメントだけ読むと何が起きてunexpectedなのかがよくわからなかったので、以下でお願いしたいです。

```suggestion
    // Closing HideShow panel by clicking again here causes runtime error (TypeError: win.PointerEvent is not a constructor), 
    // so commented here out to make tests pass for now.
```

Reply:
toggle部分だけを切り出してテストを実行見してみたら、うまく動いてくれたので、下記のように修正しました！
https://github.com/tailor-platform/frontend-packages/pull/275/commits/45cfd617b19ea8ca156bca7edeb924cecb7db27f